PaintingTrivia.py

```diff
@@ -125,35 +144,66 @@ def guess_the_painter():
             return x
 
         try:
-            chosen_painter = int(request.form['chosen_painter'])
+            quiz = pickle.loads(session['quiz'])
```

Comment:
same here. the problem is that, as it turns out, the default session mechanism in flask uses cookies to store information. that means it's stored on the client-side, what we need is a server-side storage...maybe later